app/tweet_reachout.py

```diff
@@ -0,0 +1,30 @@
+from app.twitter_bot import twitter_reply
+
+#Dummy class for a ticket to minitor replies, is not currently being used
+class TweetReachOut:
+
```

Comment:
This needs documentation. It's still early stage development so we'll revisit this later!